### PR TITLE
Merge kitchen.yml into kitchen.dokken.yml

### PR DESCRIPTION
**Pull request by limitusus**
# Description

Run tests on all platforms

## Issues Resolved

N/A

## Check List

- [x] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by limitusus** [comment]
#5 resolves ci/delivery

**Review by axelrtgs — APPROVED** [review]

**Review by axelrtgs — CHANGES_REQUESTED** [review]
I may have approved in haste. I didnt realize the kitchen file was delete. in the other repos we keep both files, one for local testing outside of ci/cd and one for testing in ci/cd.

File: kitchen.dokken.yml [hunk 1]
```diff
@@ -33,3 +33,42 @@ platforms:
...
+      image: gentoo/stage3-amd64
+      pid_one_command: /sbin/init
+      intermediate_instructions:
+        - RUN echo 'GENTOO_MIRRORS=http://gentoo.mirrors.tera-byte.com/' >> /etc/portage/make.conf
+        - RUN mkdir -p /etc/portage/repos.conf
+        - RUN cp /usr/share/portage/config/repos.conf /etc/portage/repos.conf/gentoo.conf
+        - RUN sed -i -e 's/sync-uri =.*/sync-uri = rsync:\/\/rsync.us.gentoo.org\/gentoo-portage/' /etc/portage/repos.conf/gentoo.conf
+        - RUN emerge --sync || true
+
+suites:
```
Comment (axelrtgs) [inline kitchen.dokken.yml:58]: Looking at the other repos it looks like suites is only part of the kitchen.yml and not the dokken one. i think the suites for dokken are specified somewhere else and not in the kitchen file.
EX:
https://github.com/sous-chefs/apache2/blob/master/.github/workflows/ci.yml#L50
https://github.com/sous-chefs/apache2/blob/master/kitchen.dokken.yml

Comment (limitusus) [reply kitchen.dokken.yml:58]: Here suites are necessary as run_list for the tests can only be specified here.
Having suites ci.yml is to parallelize tests by CI matrix feature.

Comment (axelrtgs) [reply kitchen.dokken.yml:58]: haven't seen it in any of the other repos but i'll trust you on this one :) as long as the tests are working i am happy

**Review by axelrtgs — COMMENTED** [review]
Is it too much trouble to ask to have a look at the gentoo tests? don't know anything about daemontools or gentoo so it would be nice to have the tests pass if it isnt too much of an ask.

**Comment by limitusus** [comment]
Fmm... I'll check it. It should not be so tough to fix gentoo tests

**Review by axelrtgs — APPROVED** [review]
Approving, changes seem to be working. Would still be nice to fix the gentoo tests but since they were already broken its not a requirement. Thanks for the contribution.

**Comment by limitusus** [comment]
Investigated that but couldn't find the cause, and a retry resolved... :)

**Pull request merged**